portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -10,9 +10,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
```

Comment:
I assume adding the space here was accidental?  There's a couple of other lines below where you've done this as well - probably want to remove that. :)

Reply:
Got it, I'll try to remember to watch for that!